### PR TITLE
Add some comments with useful info to CR

### PR DESCRIPTION
**Pull request by vsychov**
Hi guys,

docs is missing some info about configure s3 backups, I added it to CR, it may be helpful

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/init-deploy.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/one-pod.log) |
| cross-site-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/cross-site-sharded.log) |
| storage | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/storage.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/upgrade-consistency.log) |
| demand-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/demand-backup.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/self-healing.log) |
| self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/self-healing-chaos.log) |
| monitoring-2-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/monitoring-2-0.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/limits.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/scaling.log) |
| arbiter | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/arbiter.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/operator-self-healing.log) |
| operator-self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/operator-self-healing-chaos.log) |
| security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/security-context.log) |
| scheduled-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/scheduled-backup.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/service-per-pod.log) |
| rs-shard-migration | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/rs-shard-migration.log) |
| liveness | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/liveness.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/smart-update.log) |
| demand-backup-sharded | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/demand-backup-sharded.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/version-service.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/users.log) |
| data-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/data-sharded.log) |
| upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/upgrade.log) |
| non-voting | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/non-voting.log) |
| upgrade-sharded | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/upgrade-sharded.log) |
| demand-backup-eks-credentials | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/demand-backup-eks-credentials.log) |
| data-at-rest-encryption | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/data-at-rest-encryption.log) |
| pitr | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/pitr.log) |
| pitr-sharded | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1076/3efd55cc/pitr-sharded.log) |
| We run 31 out of 30|

commit: https://github.com/percona/percona-server-mongodb-operator/pull/1076/commits/3efd55cce2f7613de207c12e34bc968034866242
image: `perconalab/percona-server-mongodb-operator:PR-1076-3efd55cc`

**Comment by spron-in** [comment]
I agree that adding notes and comments to CR might help, but I would approach it in a more systematic way.
Also I'm not comfortable with having links to docs. 

Let's fix the docs for now.

**Comment by vorbidan** [comment]
Would not updating the schema in cr definition be a more systemic way? Docs are important, but schema is "importanter"... no?

File: deploy/cr.yaml [hunk 1]
```diff
@@ -427,6 +427,7 @@ spec:
 #    security:
 #      encryptionKeySecret: "my-cluster-name-mongodb-encryption-key"
 
+  # More info can be found at https://docs.percona.com/percona-backup-mongodb/index.html
   backup:
```
Comment (nmarukovich) [inline deploy/cr.yaml:431]: @vsychov could you please check @spron-in comments? 
@vorbidan we try to keep our CR up-to-date.  Did you meet any specific problems with it? What kind of changes except these comments would be useful for you?

**Review by nmarukovich — CHANGES_REQUESTED** [review]

File: deploy/cr.yaml (hunk 1 above)
Comment (vorbidan) [reply deploy/cr.yaml:431]: @nmarukovich - My comment was regarding the proposal by @vsychov to add comments in cr.yaml to describe properties and values. IMO, the CR Definition as an OpenAPI is the right place for this sort of content. 
Even though it looks tempting to add multiline comments int CR example, it seems to be counter to the goal - making it more clear.
Having a pointer to docs is OK. Maybe also a pointer to an OpenAPI for the type description?

**Pull request closed without merging**